echoflow/utils/file_utils.py

```diff
@@ -305,7 +306,7 @@ def get_zarr_list(transect_data: Union[Output, Dict], storage_options: Dict[str,
 
     return zarr_list
 
-def process_output_transects(name: str, config: Dataset, ed_list: List[Dict[str, Any]]) -> List[Output]:
+def process_output_transects(name: str, config: Dataset, stage: Stage, ed_list: List[Dict[str, Any]]) -> List[Output]:
```

Comment:
`process_output_transects`: do we want this to be `process_output_groups`?

Reply:
`process_output_groups` sounds good.

Reply:
Renamed Process Output Groups Function